### ISSUE TITLE
Toggle FusionAuth Verbiage in Simple Theme

### ISSUE THREAD

**Issue by Rodhlann** [FusionAuth/fusionauth-issues#2805]
### Problem
<img width="898" alt="image" src="https://github.com/user-attachments/assets/5c441d8b-bea3-40bb-b57b-0c9cca43ae74">

--- 

In the Simple Unauthorized template there is hard coded text "Security by FusionAuth", which is paired with a FusionAuth incidentId. This text is configurable options in the Messages section, and cannot be otherwise disabled with Simple theming. This has proven to be confusing to our users, who see the FusionAuth naming and reach out directly to FusionAuth with their issue instead of to our support team. 

### Solution
We would like the ability to either alter this message to point the users to our internal support, or have the ability to toggle the message, similar to the ability we have to toggle the FusionAuth branded floating link in the footer. 

### Alternatives/workarounds
The only workaround currently is to remove or alter the text in the Advanced Theming workflow, but because of the stability and long term support the Simple theming provides we would prefer to utilize that theming option. 

### Additional context
The FusionAuth support team asked us to open this ticket in this format so that it could be easily shared, but I will also be providing them the link so they can attach it to a ZenDesk ticket for enterprise support. 

### Community guidelines
All issues filed in this repository must abide by the [FusionAuth community guidelines](https://fusionauth.io/community/forum/topic/1000/code-of-conduct).

### How to vote
Please give us a thumbs up or thumbs down as a reaction to help us prioritize this feature. Feel free to comment if you have a particular need or comment on how this feature should work.

**Comment by daethnir** [comment]
This is both branding (as a fusionauth customer I'm paying for a product that should not be marketing itself to our customers) and end user confusion ("I'm a customer of CabbageCorp, who's this fusionauth thing, are we being MITM or hacked???")

**Comment by mooreds** [comment]
If you have a paid license this can be turned off. Go to 'styles' under the simple themes configuration, then expand the 'miscellaneous' tab. There you can toggle 'display footer' to off. 

<img width="343" alt="Screenshot 2025-01-15 at 6 42 43 AM" src="https://github.com/user-attachments/assets/0984a97c-3fbf-486e-aad9-c5151e687cdd" />

If you are using the community edition, you can't do this.

We need to document this functionality, but hopefully this helps.

**Comment by daethnir** [comment]
That option is not present for our paid FusionAuth deployment. Perhaps it's on a different paid tier?
<img width="199" alt="Screenshot 2025-01-15 at 8 49 11 AM" src="https://github.com/user-attachments/assets/7828236b-0084-4b23-8f3a-e3c0ec46bb79" />

**Comment by mooreds** [comment]
have you added your license to this deployment? https://fusionauth.io/docs/get-started/core-concepts/licensing#adding-your-license-to-your-instance has more details.

**Comment by daethnir** [comment]
@mooreds no. we hadn't. Man, what other 'obvious' steps did we miss by not having a onboarding checklist/meeting of any kind??? Much thanks.

**Comment by mooreds** [comment]
We have this doc: https://fusionauth.io/docs/get-started/download-and-install/common-configuration but maybe it isn't obvious enough.

Glad you solved the issue, sorry you ran into it.

**Comment by mooreds** [comment]
Closing as unplanned. For the community version, we'll continue to have FusionAuth branding. For the paid version it should be removed.
